package.json

```diff
@@ -7,11 +7,11 @@
     "addons-linter": "bin/addons-linter"
   },
   "scripts": {
-    "start": "node -e \"require('grunt').cli()\" null start",
```

Comment:
Does this now work if you don't have grunt-cli globally installed. That's the reason for this.